src/java.desktop/share/classes/javax/swing/JMenu.java

```diff
@@ -1256,7 +1256,11 @@ public void applyComponentOrientation(ComponentOrientation o) {
     }
 
     /**
-     * {@inheritDoc}
+     * Sets the orientation for this menu and the associated popup component
+     * determined by the <code>ComponentOrientation</code> argument.
```

Comment:
```suggestion
     * determined by the {@code ComponentOrientation} argument.
```
The new code should use javadoc syntax.

Reply:
I did not change it as then we may need to change `applyComponentOrientation` which uses the same syntax. It will look odd having different syntax in same file.